timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -64,6 +79,29 @@ public void bringAllNodesOnline() {
     }
 
     @Test
+    public void blockedLockRequestThrows503OnLeaderElection() throws InterruptedException {
```

Comment:
(if we think this is flaky i'd be fine with removing until https://github.com/palantir/atlasdb/issues/2100)